pkg/kudoctl/cmd/init/prereqs.go

```diff
@@ -54,6 +56,18 @@ func installRoleBindings(client kubernetes.Interface, opts Options) error {
 }
 
 func installNamespace(client corev1.NamespacesGetter, opts Options) error {
+	// We only manage kudo-system namespace. For others we expect they exist.
+	if opts.Namespace != defaultns {
+		_, err := client.Namespaces().Get(opts.Namespace, metav1.GetOptions{})
+		if err == nil {
+			return nil
+		}
+		if kerrors.IsNotFound(err) {
+			return fmt.Errorf("namespace %q does not exist - KUDO expects that any namespace except the default \"kudo-system\" is created beforehand", opts.Namespace)
```

Comment:
please print out the default by embedding defaultns via %s

Reply:
done